pilot/pkg/model/jwks_resolver.go

```diff
@@ -79,8 +80,8 @@ type jwtPubKeyEntry struct {
 	lastUsedTime time.Time
 }
 
-// jwksResolver is resolver for jwksURI and jwt public key.
-type jwksResolver struct {
+// JwksResolver is resolver for jwksURI and jwt public key.
+type JwksResolver struct {
```

Comment:
Can we also move `jwks_resolver.go` and `jwks_resolver_test.go` to `pilot/pkg/security/model/`? It's part of the security feature.

Reply:
Unfortunately, no.  The ParseJwksURI function in jwks_resolver.go is referenced in validation.go, and that package is referenced from Galley.

Reply:
thanks, I guess we could do a refactor later.